floris/utilities.py

```diff
@@ -260,7 +261,50 @@ def rotate_coordinates_rel_west(
         + y_center_of_rotation
     )
     z_coord_rotated = np.ones_like(wind_deviation_from_west) * z_coordinates
-    return x_coord_rotated, y_coord_rotated, z_coord_rotated
+    return x_coord_rotated, y_coord_rotated, z_coord_rotated, x_center_of_rotation, \
+        y_center_of_rotation
+
+
+def reverse_rotate_coordinates_rel_west(
```

Comment:
Please add docstring. It could be similar to the one above.

Reply:
Updated!